src/message-stream.ts

```diff
@@ -156,14 +154,27 @@ export class MessageStream extends PassThrough {
   /**
    * Destroys the stream and any underlying streams.
    *
-   * @param {error?} err An error to emit, if any.
+   * @param {error?} _error An error to emit, if any.
    * @private
    */
-  destroy(err?: Error): void {
+  destroy(_error?: Error | null): void {
```

Comment:
style nit here and below, why underscores?

Reply:
Oh, this was more or less copied out of the definitions from the Node stream library. I don't think there's a particular reason for it, so I'll move them back to no underscores.